packages/node_modules/@webex/plugin-memberships/src/memberships.js

```diff
@@ -70,6 +70,31 @@ const Memberships = WebexPlugin.extend({
 
     /* eslint-disable no-case-declarations */
     switch (activity.verb) {
+      case SDK_EVENT.INTERNAL.ACTIVITY_VERB.CREATE:
+        const membershipCreatedEventDataArray =
+          activity.object.participants.items.map(
+            (participant) => {
+              const output = cloneDeep(activity);
+
+              output.target = cloneDeep(activity.object);
+              output.object = cloneDeep(participant);
+
+              return this.getMembershipEvent(output,
+                SDK_EVENT.EXTERNAL.EVENT_TYPE.CREATED);
+            }
+          );
+
+        membershipCreatedEventDataArray.forEach(
```

Comment:
this can be on a single line

Reply:
Latest commit addresses this style concern.